fmriprep-docker error: Clobbering options: output-grid-reference

```
PS C:\Users\filo> fmriprep-docker.exe --help
Traceback (most recent call last):
  File "c:\users\filo\anaconda3\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\filo\anaconda3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\filo\Anaconda3\Scripts\fmriprep-docker.exe\__main__.py", line 9, in <module>
  File "c:\users\filo\anaconda3\lib\site-packages\fmriprep_docker.py", line 346, in main
    print(merge_help(parser.format_help(), targethelp))
  File "c:\users\filo\anaconda3\lib\site-packages\fmriprep_docker.py", line 158, in merge_help
    ', '.join(overlap - set('hvw')))
AssertionError: Clobbering options: output-grid-reference
```

--- a/.circle/tests.sh
+++ b/.circle/tests.sh
@@ -42,6 +42,7 @@ case ${CIRCLE_NODE_INDEX} in
     find ~/ds054/scratch -not -name "*.svg" -not -name "*.html" -not -name "*.svg" -not -name "*.rst" -type f -delete
     ;;
   1)
+    fmriprep-docker -i poldracklab/fmriprep:latest --help
     fmriprep-docker -i poldracklab/fmriprep:latest --config $HOME/nipype.cfg -w $HOME/ds005/scratch $HOME/data/ds005 $HOME/ds005/out participant --output-space fsaverage5 --debug --write-graph
     find ~/ds005/scratch -not -name "*.svg" -not -name "*.html" -not -name "*.svg" -not -name "*.rst" -type f -delete
     ;;

--- a/wrapper/fmriprep_docker.py
+++ b/wrapper/fmriprep_docker.py
@@ -154,8 +154,9 @@ def merge_help(wrapper_help, target_help):
 
     # Make sure we're not clobbering options we don't mean to
     overlap = set(w_flags).intersection(t_flags)
-    assert overlap == set('hvw'), "Clobbering options: {}".format(
-        ', '.join(overlap - set('hvw')))
+    expected_overlap = set(['h', 'v', 'w', 'output-grid-reference'])
+    assert overlap == expected_overlap, "Clobbering options: {}".format(
+        ', '.join(overlap - expected_overlap))
 
     sections = []
 
@@ -219,6 +220,8 @@ def main():
                         default='poldracklab/fmriprep:{}'.format(__version__),
                         help='image name')
 
+    # Options for mapping files and directories into container
+    # Update `expected_overlap` variable in merge_help() when adding to this
     g_wrap = parser.add_argument_group(
         'Wrapper options',
         'Standard options that require mapping files into the container')

